src/components/CompareResults/ResultsTable.tsx

```diff
@@ -30,45 +28,44 @@ function ResultsTable(props: ResultsTableProps) {
   const styles = {
     tableContainer: style({
       backgroundColor: themeColor100,
+      borderCollapse: 'separate',
+      borderSpacing: `0 ${Spacing.Small}px`,
       marginTop: Spacing.Large,
-      $nest: {
-        '.MuiTable-root': {
-          borderSpacing: `0 ${Spacing.Small}px`,
-          borderCollapse: 'separate',
-          paddingBottom: Spacing.Large,
-        },
-      },
+      paddingBottom: Spacing.Large,
     }),
   };
 
   return (
-    <TableContainer
-      component={Paper}
+    <Paper
       className={styles.tableContainer}
       data-testid='results-table'
       sx={customStyles}
+      role='table'
     >
       {loading ? (
         <Box display='flex' justifyContent='center' alignItems='center'>
           <CircularProgress />
         </Box>
       ) : (
-        <Table>
+        <div>
```

Comment:
Is this div useful? The `<Paper>` above already adds a container

Reply:
This `div` is useful because there is `TableHeader` component and the processedResults with `TableContent`. These components are shown based on the `loading` value.

Reply:
But you could use a fragment instead, maybe?

```js
<>
  <TableHeader />
  {processedResults...}
</>
```

Reply:
As a reminder, a fragment groups other components together when a single root is needed (like here), but otherwise doesn't render any DOM node to the page.
You might also have seen it as `<Fragment>` or `<React.Fragment>`, but `<>` is shorter.